app/src/main/java/edu/uc/reedws/musiclink/service/PlaylistService.kt

```diff
@@ -32,6 +40,10 @@ class PlaylistService(application: Application) {
         return playlist
     }
 
+    /**
+     * Save or insert the new playlist inside room database
+     *
+     */
```

Comment:
Added an appropriate comment to indicate that this function saves or inserts the new playlist inside the room database because it calls savePlaylist() method.